src/sentry/integrations/msteams/webhook.py

```diff
@@ -180,3 +199,60 @@ def handle_member_removed(self, request):
 
         integration.delete()
         return self.respond(status=204)
+
+    def handle_action_submitted(self, request):
+        data = request.data
+        channel_data = data["channelData"]
+        team_id = channel_data["team"]["id"]
+        tenant_id = channel_data["tenant"]["id"]
+        group_id = data["value"]["groupId"]
+        user_id = data["from"]["id"]
+
+        try:
+            integration = Integration.objects.get(provider=self.provider, external_id=team_id)
+        except Integration.DoesNotExist:
+            logger.info("msteams.action.missing-integration", extra={"team_id": team_id})
+            return self.respond(status=404)
+
+        try:
+            group = Group.objects.select_related("project__organization").get(
+                project__in=Project.objects.filter(
+                    organization__in=integration.organizations.all()
```

Comment:
Can integrations be linked to more than a single organization in msteams? Or does msteams follow the 1:1 model of organizations -> integraion -> installation flow that integrations like gitlab do?

Could there be a risk of cross organization data access through well crafted events to the msteams webhook endpoints?

Reply:
@markstory Great question. When we find a match for an issue, we will attempt to make a request to the issues endpoint on behalf of the linked user. That's coming in the future but we are basically copying Slack here. If the user does not have permissions for that organization, we will throw a `403` error if they try to update the issue.